src/TriggerUnit.cpp

```diff
@@ -451,18 +439,18 @@ QString TriggerUnit::assembleReport()
 
 void TriggerUnit::doCleanup()
 {
-    for(auto it = mCleanupList.begin(); it != mCleanupList.end(); it++)
+    for(auto & trigger : mCleanupList)
```

Comment:
Actually, if you hold the cursor over the item and press F1 doesn't Qt Creator tell you in some way - if you edit the item I think also a tooltip tends to pop-up, especially at the end of a meaningful point of an identifier/function telling you what it's type/what it returns...? Or is `auto` excluded from that.  I recall that this only tends to work well for data types covered by `#includes` directly in the file and is slower/not so good/doesn't work in the class implementation (<T>.cpp) file if the type definition is from a project class and the header file is pulled into the class header file (<T>.h) rather than the implementation file.